bindings/python/examples/how_tos/alias_wallet/transaction.py

```diff
@@ -41,7 +41,8 @@
     'mandatoryInputs': [input],
 }
 transaction = account.send_with_params(params, options)
-account.retry_transaction_until_included(transaction.transactionId, None, None)
+account.reissue_transaction_until_included(
+    transaction.transactionId, None, None)
```

Comment:
are these `None, None` correct? None (Heh!) of the other examples have it

Reply:
Removed